index.js

```diff
@@ -0,0 +1,143 @@
+cardValue = {
```

Comment:
You are assigning a value to a variable that doesn't exist yet. You need to use let to declare your variable first.